### PR TITLE
Default to default workspace (instead of personal workspace)

### PR DESCRIPTION
**Pull request by AidanAbd**

### CODE REVIEW

File: latch/account.py [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
 import gql
 import graphql.language as l
-from latch_sdk_config.user import user_config
 from latch_sdk_gql.execute import execute
 from latch_sdk_gql.utils import _GqlJsonValue, _json_value, _name_node, _parse_selection
 from typing_extensions import Self, TypeAlias
 
 from latch.registry.project import Project
 from latch.registry.table import Table
+from latch_cli.utils import current_workspace
```
Comment (maximsmol) [inline latch/account.py:19]: `latch` shouldn't really depend on `latch_cli` imo

File: latch_cli/utils/__init__.py [hunk 2]
```diff
@@ -12,23 +12,38 @@
...
 if "latch" not in urllib.parse.uses_netloc:
     urllib.parse.uses_netloc.append("latch")
     urllib.parse.uses_relative.append("latch")
 
 
+def get_workspaces() -> Dict[str, str]:
+    headers = {"Authorization": f"Bearer {retrieve_or_login()}"}
+
+    resp = post(
+        url=config.api.user.list_workspaces,
```
Comment (maximsmol) [inline latch_cli/utils/__init__.py:39]: this should be GQL now

File: latch_cli/utils/__init__.py [hunk 3]
```diff
@@ -99,7 +114,24 @@ def current_workspace() -> str:
...
+        )["userInfoByAccountId"]["defaultAccount"]
+
+        workspace_names = get_workspaces()
+
+        if ws not in workspace_names:
+            raise ValueError(
+                f"Default workspace {ws} not found in the list of workspaces."
+            )
+
+        user_config.update_workspace(ws, workspace_names[ws])
```
Comment (maximsmol) [inline latch_cli/utils/__init__.py:134]: default workspace should auto-update unless you explicitly overwrote it

so i'd just remove it

File: latch_cli/utils/__init__.py [hunk 4]
```diff
@@ -99,7 +114,24 @@ def current_workspace() -> str:
     ws = user_config.workspace_id
     if ws == "":
         ws = account_id_from_token(retrieve_or_login())
-        user_config.update_workspace(ws, "Personal Workspace")
+        ws = execute(
+            gql.gql("""
+            query GetUserDefaultWorkspace($accountId: BigInt!) {
+                userInfoByAccountId(accountId: $accountId) {
+                    defaultAccount
```
Comment (maximsmol) [inline latch_cli/utils/__init__.py:121]: might be unset - you really want to default to current account if it's null

**Review by maximsmol — CHANGES_REQUESTED** [review]

File: latch/utils.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+            teamMembers(filter: {user: {accountId: {equalTo: $accountId}}}) {
+                nodes {
+                    team {
+                        accountId
+                        displayName
+                    }
+                }
+            }
+        }"""),
+        {"accountId": account_id},
```
Comment (ayushkamat) [inline latch/utils.py:67]: can u do this using `accountInfoCurrent` somehow instead of passing in `account_id`

**Review by ayushkamat — CHANGES_REQUESTED** [review]

File: latch/utils.py (hunk 5 above)
Comment (AidanAbd) [reply latch/utils.py:67]: Unfortunately not.

**Pull request merged**